jackson/src/main/java/io/gravitee/definition/jackson/datatype/services/core/deser/ScheduledServiceDeserializer.java

```diff
@@ -39,43 +41,61 @@ public ScheduledServiceDeserializer(Class<T> vc) {
     protected void deserialize(T service, JsonParser jsonParser, JsonNode node, DeserializationContext ctxt) throws IOException, JsonProcessingException {
         super.deserialize(service, jsonParser, node, ctxt);
 
-        // New version of scheduler service
-        final JsonNode triggerNode = node.get("trigger");
 
-        if (triggerNode != null) {
-            Trigger trigger = new Trigger();
-            final JsonNode rateNode = triggerNode.get("rate");
-
-            if (rateNode != null) {
-                trigger.setRate(rateNode.asLong());
-            } else {
-                throw ctxt.mappingException("[scheduled-service] Rate is required");
-            }
-
-            final JsonNode unitNode = triggerNode.get("unit");
-            if (unitNode != null) {
-                trigger.setUnit(TimeUnit.valueOf(unitNode.asText().toUpperCase()));
-            } else {
-                throw ctxt.mappingException("[scheduled-service] Unit is required");
-            }
-            service.setTrigger(trigger);
+        final JsonNode scheduleNode = node.get("schedule");
+        String schedule = null;
```

Comment:
you can remove this useless initializer
```suggestion
        String schedule;
```